server/public/css/main.css

```diff
@@ -487,4 +487,46 @@ ul,li,body{padding: 0;margin: 0;}
     position:absolute;
     top:100px;
     right:10px;
-}
\ No newline at end of file
+}
+
+.more-tjm-info{
+    width:60px;
+    float: right;
+    margin-right:10px;
+}
+
+.white_content {
+    display: none;
+    position: absolute;
+    top: 18%;
+    left: 30%;
+ 
+    width: 400px;
+    height: 500px;
+    background: #FFFFFF;
+    border-radius: 6px;
+ 
+    z-index: 1002; 
+    overflow: auto;
+    position: fixed;
+}
+ 
+.top-div {
+    display: flex;
+    height: 56px;
+    background: #FFFFFF;
+    border-bottom: 1px solid #cccc;;
+}
+ 
+.closeBtn{
```

Comment:
can we try to use bootstrap buttons, this will have a single style as opposed to multiple buttons